packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -34,9 +34,6 @@ library TaikoData {
         uint32 blockFeeBaseGas;
```

Comment:
Not related to this PR but seems this config is no longer used?

Reply:
removed